spring-integration-sftp/src/main/java/org/springframework/integration/sftp/dsl/Sftp.java

```diff
@@ -122,13 +124,36 @@ public static SftpMessageHandlerSpec outboundAdapter(RemoteFileTemplate<ChannelS
 	 * @param remoteFileTemplate the remote file template.
 	 * @param fileExistsMode the file exists mode.
 	 * @return the spec.
+	 * @deprecated in favor of {@link #outboundAdapter(SftpRemoteFileTemplate,FileExistsMode)}
 	 */
+	@Deprecated
 	public static SftpMessageHandlerSpec outboundAdapter(RemoteFileTemplate<ChannelSftp.LsEntry> remoteFileTemplate,
 			FileExistsMode fileExistsMode) {
 
 		return new SftpMessageHandlerSpec(remoteFileTemplate, fileExistsMode);
 	}
 
+	/**
+	 * An {@link SftpMessageHandlerSpec} factory for an outbound channel adapter spec.
+	 * @param sftpRemoteFileTemplate the remote file template.
+     * @return the spec.
```

Comment:
Please, format JavaDocs properly.
And of course `@since 5.4`!

Reply:
Added